docs/topics/async.txt

```diff
@@ -0,0 +1,52 @@
+====================
+Asynchronous Support
+====================
+
+Django has developing support for asynchronous ("async") Python, but does not yet
```

Comment:
Did you mean "Django **is** developing support..." or "Django has **developed** support..."?

Reply:
I meant "has developing", as written - it has "developing support".